packages/compadre/examples/Compadre_UnitTests.cpp

```diff
@@ -16,7 +16,7 @@ ASSERT_EXIT({{ statement } ::exit(EXIT_SUCCESS); }, ::testing::ExitedWithCode(0)
 // KokkosParser tests go here because they modify the Kokkos backend
 //
 TEST (KokkosInitialize, NoArgsGiven) { 
-    Kokkos::InitArguments args;
+    Kokkos::InitializationSettings args;
```

Comment:
Below (L29) using deprecated `--kokkos-threads` command line argument

Reply:
Thanks for the catch, will fix

Reply:
Fixed with fe70220143d5d4c24d4c0a491c7bbe4c7fee442c

Reply:
Chatted about compadre changes with @kuberry , these will be reverted and submitted separately following merge of this PR